examples/collectionless-item.json

```diff
@@ -74,24 +75,34 @@
   "links": [
     {
       "rel": "self",
-      "href": "http://cool-sat.com/catalog/CS3-20160503_132130_04/CS3-20160503_132130_04.json"
+      "href": "https://raw.githubusercontent.com/radiantearth/stac-spec/examples-refresh/examples/simple-item.json",
+      "type": "application/geo+json; profile=stac-item"
```

Comment:
The media types are outdated

Reply:
Thanks - I tried to catch all of them, but forgot to do an extensive review.